### ISSUE TITLE
Request failed :java.lang.IllegalArgumentException

### ISSUE THREAD

**Issue by 2delarosa** [sivasamyk/graylog2-plugin-input-httpmonitor#11]
I'm running graylog-3.2.4 and I downloaded graylog2-plugin-input-httpmonitor-1.0.5.jar to the plugin folder.  I create a GET HTTP reguest but I'm getting the following error:
```
Request failed :java.lang.IllegalArgumentException: Header name cannot contain the following prohibited characters: =,;: \t\r\n\v\f
```
I don't believe I'm using illegal characters.  Do you have any suggestions on how to troubeshoot?

Thanks in advance.

HTTP Monitor Input Configuration
```
configHeadersToRecord: <empty>
configHeadersToSend: Accept:application/json, Authorization:Bearer d017317b89c07500fcc9a43074d038e34c4a3aa0248de9e419d8a10b80bea65e
configHttpProxy: <empty>
configInterval: 1
configIntervalUnit: MINUTES
configLabel: A3RegisteredNodes
configLogResponseBody: true
configMethod: GET
configPassword:********
configRequestBody: <empty>
configTimeout: 20
configTimeoutUnit: SECONDS
configURL: https://192.168.128.248:9999/api/v1/reports/registered
configUsername: <empty>
decompress_size_limit: 8388608
override_source: <empty>
```

**Comment by jeremyglanum** [comment]
Hi, 

Do you have any news about compatibility with graylog V3 ? 

Thanks,

**Comment by sivasamyk** [comment]
@jeremyglanum Are you getting similar error? I have not verified the plugin with graylog v3 yet.

**Comment by jeremyglanum** [comment]
@sivasamyk, No, I havent error, I was just afraid about the compatibility but all works with the V3. 

Thanks,

**Comment by sivasamyk** [comment]
@2delarosa Can you update your header config from
`Accept:application/json, Authorization:Bearer d017317b89c07500fcc9a43074d038e34c4a3aa0248de9e419d8a10b80bea65e` to `Accept:application/json,Authorization:Bearer d017317b89c07500fcc9a43074d038e34c4a3aa0248de9e419d8a10b80bea65e`? Notice that I have removed the space before `Authorization:...`. I tested without space and it works fine.

@jeremyglanum I verified this plugin with Graylog 3.2.4 and it works fine!
